politeiawww/user.go

```diff
@@ -200,6 +205,56 @@ func validatePubkey(publicKey string) ([]byte, error) {
 	return pk, nil
 }
 
+// formatName normalizes a contractor name to lowercase without leading and
+// trailing spaces.
+func formatName(name string) string {
+	return strings.ToLower(strings.TrimSpace(name))
+}
+
+func validateName(name string) error {
```

Comment:
The current name regex doesn't allow for spaces in the name.  Is this intended?

Also, shouldn't this be returning a `MalformedName` error and not a `MalformedUsername` error?